### PR TITLE
Separate pod for database

### PR DESCRIPTION
**Pull request by skanthed**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -70,4 +70,8 @@ while true; do
         month ) cp scripts/reports/month_report_cronjob.yaml scripts/report_cronjob.yaml; break;;
         * ) echo "Please answer day/week/month.";;
     esac
-done
\ No newline at end of file
+done
+
+cp Documentation/config/config.env database/config/config.env
+cd database
+kustomize build | oc apply -f-
```
Comment (leihchen) [inline install.sh:77]: `kustomize build database | oc apply -f-`
delete `cd database`

File: database/scripts/schema.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import psycopg2, time, os
```
Comment (leihchen) [inline database/scripts/schema.py:1]: Using python psycopg2 client is unnecessary. 
Just use psql: 
``` sh
psql -d "postgresql://$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST_NAME:$PORT_NUMBER/$DATABASE_NAME" -f generate_report.psql
```

File: database/cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: database-execution
+spec:
+  volumes:
+  - name: koku-metrics-operator-data
+    persistentVolumeClaim:
+      claimName: koku-metrics-operator-data
```
Comment (leihchen) [inline database/cronjob.yaml:9]: Remove unnecessary PVC

File: database/cronjob.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    - configMapRef:
+        name: backup-config
+    command:
+    - python3
+    - "/scripts/schema.py"
+    volumeMounts:
+    - name: backup-scripts
+      mountPath: "/scripts"
+    - name: koku-metrics-operator-data
+      mountPath: "/tmp/koku-metrics-operator-data"
```
Comment (leihchen) [inline database/cronjob.yaml:26]: Remove unnecessary mount

File: database/scripts/schema.py [hunk 5]
```diff
@@ -21,13 +19,10 @@ def execute_sql_commands(command):
...
         
-        cursor.execute(command)
-
-        # close communication with the PostgreSQL database server
-        cursor.close()
-
-        # commit the changes
+        cursor.execute(command)  # close communication with the PostgreSQL database server
+        cursor.close()  # commit the changes
         conn.commit()
```
Comment (leihchen) [inline database/scripts/schema.py:24]: Mistake here.

**Review by leihchen — APPROVED** [review]

File: database/scripts/schema.py (hunk 2 above)
Comment (skanthed) [reply database/scripts/schema.py:1]: Python or shell it's not a big deal. If we want to make or add changes later.

**Pull request merged**